src/model/events/content-types.ts

```diff
@@ -94,6 +94,8 @@ const mimeTypeToContentTypeMap: { [mimeType: string]: ViewableContentType } = {
     'application/vnd.google.protobuf': 'protobuf',
     'application/x-google-protobuf': 'protobuf',
     'application/proto': 'protobuf', // N.b. this covers all application/XXX+proto values
+    'application/grpc': 'protobuf', // Used in GRPC requests
+    'application/x-protobuffer': 'protobuf' // Commonly seen in Google apps
```

Comment:
Oh, also there's an error in the build because there's a missing `,` at the end here